practices/0005-team-growth-tracking.md

```diff
@@ -0,0 +1,80 @@
+# Team Growth Tracking
+
+We believe strongly in the growth of our team members. Expanding our
+knowledge and cohesiveness is a constant aspiration. With that goal in mind,
+we have created a GitHub Project and Epics to help track our effort in
+expanding our team skills both technical and personal. We have also created
+GitHub issue [templates][1] to speed up the process of submitting a team
+growth issue.
+
+[1]:https://github.com/CCI-MOC/ops-private/issues/templates/edit
+
```

Comment:
Just a comment; not a request for a change:

For a document like this, it often makes more sense to use named rather than numbered links. That is, rather than:

```
GitHub issue [templates][1] to speed up the process of submitting a team

[1]: https://github.com/CCI-MOC/ops-private/issues/templates/edit
```

You can do this if the link name is the same as the link title:

```
GitHub issue [templates][] to speed up the process of submitting a team

[templates]: https://github.com/CCI-MOC/ops-private/issues/templates/edit
```

Or you can do something like this:

```
GitHub issue [templates][foo] to speed up the process of submitting a team

[foo]: https://github.com/CCI-MOC/ops-private/issues/templates/edit
```

This makes life easier as you add or re-arrange links in the document.